package mg.gestionsalle.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "salle")
public class Salle {

    @Id
    @Column(name = "codesal")
    private String codesal;

    @Column(name = "designation")
    private String designation;

    public Salle() {
    }

    public Salle(String codesal, String designation) {
        this.codesal = codesal;
        this.designation = designation;
    }

    public String getCodesal() {
        return codesal;
    }

    public void setCodesal(String codesal) {
        this.codesal = codesal;
    }

    public String getDesignation() {
        return designation;
    }

    public void setDesignation(String designation) {
        this.designation = designation;
    }
}